docs/pages/access-controls/guides/device-trust.mdx

```diff
@@ -170,8 +223,8 @@ A registered device becomes a trusted device after it goes through the
 enrollment ceremony. Enrollment exchanges an enrollment token, created by a
 device admin, for the opportunity to enroll the corresponding device.
 
-macOS enrollments are tied to the current user in the device. If the same device
-is handed to a new user, they will need to perform enrollment again.
+Enrollments are tied to the current user in the device. If the same device is
```

Comment:
```suggestion
Enrollments are tied to the current user in the device. If the same device is
```
maybe something like:
Device enrollment is specifically linked to the user who performed the enrollment. If the same device is handed to a new user, that user must enroll the device again.

Reply:
Yeah, that's nicer. Thanks.